decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -285,11 +303,9 @@
   factory :newsletter, class: "Decidim::Newsletter" do
     author { build(:user, :confirmed, organization: organization) }
     organization
-
-    # rubocop:disable RSpec/EmptyLineAfterSubject
     # Bug in rubocop-rspec
```

Comment:
Remove this line too, since this is fixed now 😄

Reply:
:+1: